crates/bevy_asset/src/loader.rs

```diff
@@ -39,6 +39,10 @@ pub trait AssetLoader: Send + Sync + 'static {
 
     /// Returns a list of extensions supported by this asset loader, without the preceding dot.
     fn extensions(&self) -> &[&str];
+    /// Returns the type name of the sub-asset type with the given `label`.
+    fn label_type_name(&self, label_type: &str) -> Option<&'static str>;
```

Comment:
I think the new methods should have default implementations returning name/I'd of `Self::Asset`

Reply:
That should make this change non breaking.

Reply:
I was on the fence yeah. I kinda wanted to make the implementation for the user as easy as possible, and parsing a label is kinda annoying, but yeah in doing that I kinda set in stone what a label should look like and we might not want that.

I would like to hear another opinion on this.